src/Vlingo.UUID.Tests/Vlingo.UUID.Tests.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp3.0</TargetFramework>
     <LangVersion>latest</LangVersion>
+    <Configurations>Debug;Release;Debug With Project References</Configurations>
```

Comment:
This is only used by the platform right ?

Reply:
Yes

Reply:
Yes